src/dataDisplay/Identicon/index.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+
+import makeBlockie from 'ethereum-blockies-base64';
+import { RefObject, useEffect, useState } from 'react';
+
+type Props = {
+  address: string;
+  diameter: number;
```

Comment:
Diameter values should be handled through the theme, as we do for Text size for example.